### ISSUE TITLE
Very slow multi-user sessions on windows

### ISSUE THREAD

**Issue by 60-hz** [cables-gl/cables_docs#902]
**Link to simple, reproducible example patch**
Any patch.

**Describe the bug**
Under Windows, (Chrome or firefox browser),  joining a patch in editor mode while another user is connected and active drop the performance drastically and make the patch almost unusable. When 2nd user is not active anymore, performance go back to normal.

Same issue in dev.cables.gl
No issue on MacOS with same browser / patch.

**How To Reproduce**
1. Open a patch in editor mode in computer 1
2. Open same patch in editor mode in computer 2
3. Slowdown / CPU spike on windows side when another user is active.

**Platform**
 - Windows10, Chrome last version :  130.0.6723.117
 - Windows10, Firefox 132.0.1

**Comment by steam0r** [comment]
i just tried to reproduce this (on window 11, though) and am not experiencing this...can you try this on another machine, or maybe a faster network connection and then re-report...thank you!

**Comment by 60-hz** [comment]
Yes, I post this issue because I experience the same in school computer's room. Few of them are still under Windows10.
But since I see a spike in cpu, I guess it not related to network?
